examples/list_box_model/list_box_row/imp.rs

```diff
@@ -0,0 +1,154 @@
+use std::cell::RefCell;
+
+use gtk::{
+    glib::{self, clone, ParamSpec, ParamSpecObject, Value},
+    prelude::*,
+    subclass::prelude::*,
+    ResponseType,
+};
+
+use crate::row_data::RowData;
+
+#[derive(Default, Debug)]
+pub struct ListBoxRow {
+    row_data: RefCell<Option<RowData>>,
+    parent_window: RefCell<Option<gtk::Window>>,
+}
+
+#[glib::object_subclass]
+impl ObjectSubclass for ListBoxRow {
+    const NAME: &'static str = "ExListBoxRow";
+    type ParentType = gtk::ListBoxRow;
+    type Type = super::ListBoxRow;
+}
+
+impl ObjectImpl for ListBoxRow {
+    fn properties() -> &'static [ParamSpec] {
+        use once_cell::sync::Lazy;
+        static PROPERTIES: Lazy<Vec<ParamSpec>> = Lazy::new(|| {
+            vec![
+                ParamSpecObject::new(
+                    "rowdata",
+                    "RowData",
+                    "RowData",
+                    RowData::static_type(),
+                    glib::ParamFlags::READWRITE | glib::ParamFlags::CONSTRUCT,
```

Comment:
```suggestion
                    glib::ParamFlags::READWRITE | glib::ParamFlags::CONSTRUCT_ONLY,
```

Maybe? Can it be changed at a later time and that actually has an effect?

Reply:
I think that should be fine, I never call `set_property` myself so that should only be set by glib/gtk I think